### DISCUSSION TITLE
Is there some repos about the paper "Compressing deep neural networks on FPGAs to binary and ternary precision with hls4ml"

### DISCUSSION THREAD

**Discussion by lpj20** [fastmachinelearning/hls4ml#638] [Q&A]
Is there some repos about the paper "Compressing deep neural networks on FPGAs to binary and ternary precision with hls4ml"

**Comment by sandeep1404** [comment]
HI, if you can able to find the repo please share here even I am looking for it.

**Reply by vloncar** [reply]
From the paper itself: "The implementation of these solutions is integrated in recent versions of the hls4ml library, starting with the v0.1.6
tag of the GitHub repository."

Note that this functionality has been superseded by the support for QKeras `binary` and `ternary` quantizers in current version of hls4ml and the quoted version is now old and unsupported.
